client/src/components/PublicData/PublicData.js

```diff
@@ -94,38 +94,50 @@ const PublicDataListing = ({ canDownload, downloadCallback }) => {
   }, [path, queryString]);
 
   return (
-    <div styleName="container">
-      <DataFilesSearchbar
-        api="tapis"
-        scheme="public"
-        system={system}
-        publicData
-      />
-      <div styleName="header">
+    /* !!!: Temporary bad indentation to make simpler PR diff */
+    /* eslint-disable prettier/prettier */
+    <Section
+      // HACK: Replicate wrapper class gives button correct global style
+      // WARNING: Applies unused and redundant `.workbench-content` styles
+      className="workbench-content"
+      header={
         <DataFilesBreadcrumbs
-          styleName="header-title"
           api={api}
           scheme={scheme}
           system={system}
           path={path || ''}
           section="FilesListing"
           isPublic
         />
+      }
+      headerActions={
         <ToolbarButton
           text="Download"
           iconName="download"
           onClick={downloadCallback}
           disabled={!canDownload}
         />
-      </div>
+      }
+    >
+      <SectionTable styleName="content" manualContent>
```

Comment:
__Reminder__
[Whenever using `manualContent` for a table, one __must__ add a `<div className="o-flex-item-table-wrap">`.](https://github.com/TACC/Core-Portal/blob/06cb7186e3cf13608071152a936177f472b0669f/client/src/components/_common/SectionTable/SectionTable.js#L46-L66)

__Explanation__
The `manualContent` prop allows injection of `{!modal && <DataFilesSearchbar />}` **outside** the `<div className="o-flex-item-table-wrap">`.